tests/unittests/graphTest.cpp

```diff
@@ -1154,6 +1154,7 @@ TEST(Graph, verifyConstantNoWriters) {
 
   EXPECT_DEATH(M.verify(), "");
 }
+#endif /* NDEBUG */
```

Comment:
Yes, because I need to roll up the refactoring to the function and the module.
I didn't want to do it in one go in case the general direction was not good.
Will post PRs to address that soon-ish ;)